lib/screens/mobile/mobile_home.dart

```diff
@@ -617,10 +620,16 @@ class _MobileHomeScaffoldState extends State<MobileHomeScaffold> with TickerProv
               );
             }
           },
-          color: Colors.blueAccent,
-          minWidth: 200,
-          shape: RoundedRectangleBorder(
-            borderRadius: BorderRadius.circular(8),
+          style: const ButtonStyle(
+            backgroundColor: MaterialStatePropertyAll(Colors.blue),
+            shape: MaterialStatePropertyAll(
```

Comment:
`MaterialStatePropertyAll` is deprecated and shouldn't be used. Use `WidgetStatePropertyAll` instead.